index.html

```diff
@@ -365,6 +375,67 @@ <h3><dfn>Quad</dfn> interface</h3>
     </p>
   </section>
 
+  <section data-dfn-for="QuadTerm">
+    <h3><dfn>QuadTerm</dfn> interface</h3>
+
+    <pre class="idl">
+    interface QuadTerm : Quad {
+      attribute string termType;
+      attribute string? value;
```

Comment:
Can we make this non-optional? We need predictable objects for high performance.

Reply:
I changed it in #165. The approach in #165 got more positive feedback than this one. If you have good arguments to continue with this PR, I can change it, but first, it would be good to know if it's worth the effort.

Reply:
Just trying to understand/compare the different options (following https://github.com/rdfjs/data-model-spec/pull/163/files#r455358833).

So I will review this one negatively for this issue; that might help us with selecting. (i.e., I could still champion this one if the optional issue is resolved)